### PR TITLE
Fix #8217: Remove Cookie Consent info popup.

### LINKED ISSUE #8217: Remove cookie consent info popup
platform parity

### PR DESCRIPTION
**Pull request by iccub**
## Summary of Changes

This pull request fixes #8217 

## Submitter Checklist:

- [x] *Unit Tests* are updated to cover new or changed functionality
- [x] User-facing strings use `NSLocalizableString()`
- [x] New or updated UI has been tested across:
  - [x] Light & dark mode
  - [x] Different size classes (iPhone, landscape, iPad)
  - [x] Different dynamic type sizes

## Test Plan:

Verify that the cookie consent popup does not show anymore.
Verify that cookie consent are still being blocked.

## Reviewer Checklist:

- [x] Issues include necessary QA labels:
  - `QA/(Yes|No)`
  - `bug` / `enhancement`
- [x] Necessary [security reviews](https://github.com/brave/security/issues/new/choose) have taken place.
- [x] Adequate unit test coverage exists to prevent regressions.
- [x] Adequate test plan exists for QA to validate (if applicable).
- [x] Issue and pull request is assigned to a milestone (should happen at merge time).

### CODE REVIEW

File: Sources/BraveStrings/BraveStrings.swift [hunk 1]
```diff
@@ -4340,6 +4340,12 @@ extension Strings {
     value: "No Saved Website Data",
     comment: "Shown when the user has no website data (such as cookies, caches, or local storage) persisted to their device."
   )
+  
+  public static let blockCookieConsentNotices = NSLocalizedString(
+      "BlockCookieConsentNotices",
+      tableName: "BraveShared", bundle: .module,
+      value: "Block Cookie Consent Notices",
+      comment: "A title for a setting that enables cookie consent notices")
```
Comment (cuba) [inline Sources/BraveStrings/BraveStrings.swift:4348]: Do we need this?

**Review by cuba — APPROVED** [review]
I think you have added back some old translation by accident. Approving this but make sure to remove this if it's not needed.

File: Sources/BraveStrings/BraveStrings.swift (hunk 1 above)
Comment (iccub) [reply Sources/BraveStrings/BraveStrings.swift:4348]: yes its used in the shields view as well

File: Sources/Onboarding/ProductNotifications/FullScreenCalloutManager.swift [hunk 2]
```diff
@@ -22,7 +22,7 @@ public enum FullScreenCalloutType: CaseIterable {
    - Cookie Notification
    - VPN Link Receipt
   */
-  case p3a, vpnUpdateBilling, bottomBar, vpnPromotion, defaultBrowser, rewards, blockCookieConsentNotices, vpnLinkReceipt
```
Comment (soner-yuksel) [inline Sources/Onboarding/ProductNotifications/FullScreenCalloutManager.swift:25]: nit: same can you update the comment

File: Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift [hunk 3]
```diff
@@ -53,8 +53,6 @@ extension BrowserViewController {
       presentDefaultBrowserScreenCallout()
     case .rewards:
       presentBraveRewardsScreenCallout(skipSafeGuards: skipSafeGuards)
-    case .blockCookieConsentNotices:
```
Comment (soner-yuksel) [inline Sources/Brave/Frontend/Browser/BrowserViewController/BrowserViewController+Callout.swift:56]: nit: can you update the comment of this function

**Review by soner-yuksel — APPROVED** [review]

**Pull request merged**